test/blackbox-tests/test-cases/ppx-cross-context-issue.t/run.t

```diff
@@ -0,0 +1 @@
+  $ dune build --debug-dependency-path
```

Comment:
Did you forget to promote?

This test also needs a description of the problem that it's trying to test.

Reply:
Promoted the error and added a description!